app/src/main/AndroidManifest.xml

```diff
@@ -37,7 +37,7 @@
             android:allowBackup="false"
             android:usesCleartextTraffic="true"
             android:supportsRtl="false"
-            tools:replace="android:supportsRtl"
+            tools:replace="android:supportsRtl,android:allowBackup"
```

Comment:
This was to fix the following error:
```
Manifest merger failed : Attribute application@fullBackupContent value=(true)
```
It's was suggested by [the documentation](https://dev.appsflyer.com/hc/docs/install-android-sdk#meta-install-referrer).